### DISCUSSION TITLE
Users reset passwords

### DISCUSSION THREAD

**Discussion by charlotattardjpj** [drakkan/sftpgo#459] [Ideas]
Is it possible for users to reset passwords themselves without the intervention of an admin. ?

Thanks a lot

**Comment by drakkan** [comment]
Hi,

yes, users can change their credentials using the [web client UI](https://github.com/drakkan/sftpgo/blob/main/docs/web-client.md).

Please be sure to not deny `HTTP` protocol, otherwise users cannot use the web client ui

**Comment by charlotattardjpj** [comment]
Thanks a lot for your very fast replies. I really appreciate your work.

The url http://myhost:8080/web/client returns The page you are looking for does not exist. The http://myhost:8080/web/users is available but only admins can login through it. 

Is it because the HTTP is denied ? if so how can I enable it.

**Reply by drakkan** [reply]
Are you using v2.1.0? The web client UI is not available in previous versions. If you are already using v2.1.0 please post the startup logs, thank you

**Comment by charlotattardjpj** [comment]
No im using an older version. 2.0.4. 

Will install the new version. Thankyou
